aws/rds_enhanced_monitoring/lambda_function.py

```diff
@@ -233,9 +254,11 @@ def lambda_handler(event, context):
     log_events = event["logEvents"]
 
     for log_event in log_events:
-        message = json.loads(log_event["message"])
+        messages = extract_json_objects(log_event["message"])
         ts = log_event["timestamp"] / 1000
-        _process_rds_enhanced_monitoring_message(ts, message, account, region)
+        for json_object in messages:
+            message = json.loads(json_object)
+            _process_rds_enhanced_monitoring_message(ts, message, account, region)
```

Comment:
Could we try to json.loads each of the messages and confirm that works before calling process? And fallback to assuming we have a single json msg in the case of failing to json load any of the sub-messages? That way if we get some message that we miss-parse, we'll fallback to the old method in case it's a valid single object message.

Reply:
Done

Reply:
A complete try/except is a little scary - if we have an issue sending to datadog intake, or any other similar issue we'll double our calls this way. Maybe just accept json.loads errors? If we mis-parse a single message into multiple messages I think it will be the case that json loads fails.